<launch>

  <node pkg="fixed_hk_nosdk" type="obs_node.py" name="obs_client_node" respawn="false" output="screen">
      <param name="camera_id" type="int" value="1"/>
      <param name="device_ip" value="192.168.1.64"/>
      <param name="device_port" value="9999"/>
      <param name="image_width" type="int" value="160"/>
      <param name="image_height" type="int" value="120"/>
  </node>
  
</launch>


